src/atoms/Checkbox/CheckboxIcon.native.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I thought it would be clean to keep `<CheckboxIcon />` as separate file. Otherwise, I will have to write the conditional check to render `<CheckedIcon />` or `<UncheckedIcon>` in `Checkbox.native.js` file